airbyte-workers/build.gradle

```diff
@@ -19,21 +19,24 @@ dependencies {
     implementation 'org.eclipse.jetty:jetty-server:9.4.31.v20200723'
     implementation 'org.eclipse.jetty:jetty-servlet:9.4.31.v20200723'
 
-
+    implementation project(':airbyte-analytics')
     implementation project(':airbyte-api')
     implementation project(':airbyte-config:models')
     implementation project(':airbyte-config:persistence')
     implementation project(':airbyte-db:lib')
     implementation project(':airbyte-json-validation')
     implementation project(':airbyte-protocol:models')
     implementation project(':airbyte-scheduler:persistence')
+    implementation project(':airbyte-scheduler:models')
 
     testImplementation 'org.mockito:mockito-inline:4.0.0'
     testImplementation 'org.postgresql:postgresql:42.2.18'
     testImplementation "org.flywaydb:flyway-core:7.14.0"
     testImplementation 'io.temporal:temporal-testing:1.6.0'
     testImplementation 'org.testcontainers:testcontainers:1.15.3'
     testImplementation 'org.testcontainers:postgresql:1.15.3'
+    testImplementation 'io.temporal:temporal-testing-junit5:1.5.0'
```

Comment:
nit: sort alphabetically (although I know this was already not correctly sorted when you started :))

Reply:
Done + fix version and remove duplicate.